app/scripts/templates/sign_in.mustache

```diff
@@ -19,8 +19,13 @@
     {{^error}}
       <div class="error"></div>
       <div class="success"></div>
+      
+      {{#showSyncSuggestion}}
+        <div class="info nudge" id="suggest-sync">{{#t}}Looking for Firefox Sync? <a href="https://mozilla.org/firefox/sync">Get started here.</a>{{/t}}</div>
+      {{/showSyncSuggestion}}
+
       {{#isSyncMigration}}
-          <div class="info nudge">{{#t}}Migrate your sync data by signing in to your Firefox&nbsp;Account.{{/t}}</div>
+          <div class="info nudge migration">{{#t}}Migrate your sync data by signing in to your Firefox&nbsp;Account.{{/t}}</div>
```

Comment:
@larissagaulia I know why its used, I am asking why its a class while `suggest-sync` is an id. Consistency would be better